ocaml/idl/datamodel.ml

```diff
@@ -9209,6 +9211,61 @@ let alert =
     ()
 *)
 
+
+(** network sriov **)
+module Network_sriov = struct
+  let lifecycle = [Published, rel_kolkata, ""]
+
+  let configuration_mode = Enum ("configuration_mode",
+    [
+      "sysfs", "Configure network sriov by sysfs, do not need reboot";
+      "modprobe", "Configure network sriov by modbrope, need reboot";
+      "unknown", "Unknown mode";
+    ])
+
+  let create = call
+      ~name:"create"
+      ~doc:"Create a network-sriov based on the specific PIF, it will automatically create a logical PIF to connect the specific network."
```

Comment:
A little more explanation about the purpose of this would be useful. I.e. that this is to enable SR-IOV on a particular PIF.

Reply:
yes